modules/chanserv/cs_drop.cpp

```diff
@@ -58,7 +58,7 @@ class CommandCSDrop final
 				*code = Anope::Random(15);
 			}
 
-			source.Reply(_("Please confirm that you want to drop \002%s\002 with with \002DROP %s %s\002."),
+			source.Reply(_("Please confirm that you want to drop \002%s\002 with \002DROP %s %s\002."),
```

Comment:
```suggestion
			source.Reply(_("Please confirm that you want to drop \002%s\002 with \002DROP %s %s\002"),
```

We may as well do this at the same time.